src/components/SiteFooter/index.jsx

```diff
@@ -0,0 +1,214 @@
+                src="https://prod-edxapp.edx-cdn.org/static/images/app/google_play_badge_45.6ea466e328da.png"
+              />
+            </a>
+          </li>
+        </ul>
+      );
+    }
+    return mobileLinks;
+  }
+
+  render() {
+    const {
+      siteName,
+      openSourceUrl,
+      termsOfServiceUrl,
+      privacyPolicyUrl,
+      contactUrl,
+      supportUrl,
+    } = this.props;
+    return (
+      <footer
+        role="contentinfo"
+        aria-label="Page Footer"
+        className="footer d-flex justify-content-center border-top py-3 px-4"
+      >
+        <div className="max-width-1180 d-grid">
+          <div className="area-1">
+            <Hyperlink destination={this.renderMarketingSiteUrl('/')} content={this.renderSiteLogo()} aria-label={`${siteName} Home`} />
+          </div>
+          <div className="area-2">
+            <h2>{siteName}</h2>
+            <ul className="list-unstyled p-0 m-0">
+              <li><a href={this.renderMarketingSiteUrl('/about-us')}>About</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/enterprise')}>{siteName} for Business</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/affiliate-program')}>Affiliates</a></li>
+              <li><a href={openSourceUrl}>Open {siteName}</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/careers')}>Careers</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/news-announcements')}>News</a></li>
+            </ul>
+          </div>
+          <div className="area-3">
+            <h2>Legal</h2>
+            <ul className="list-unstyled p-0 m-0">
+              <li><a href={termsOfServiceUrl}>Terms of Service &amp; Honor Code</a></li>
+              <li><a href={privacyPolicyUrl}>Privacy Policy</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/accessibility')}>Accessibility Policy</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/trademarks')}>Trademark Policy</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/sitemap')}>Sitemap</a></li>
+            </ul>
+          </div>
+          <div className="area-4">
+            <h2>Connect</h2>
+            <ul className="list-unstyled p-0 m-0">
+              <li><a href={this.renderMarketingSiteUrl('/blog')}>Blog</a></li>
+              <li><a href={contactUrl}>Contact Us</a></li>
+              <li><a href={supportUrl}>Help Center</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/media-kit')}>Media Kit</a></li>
+              <li><a href={this.renderMarketingSiteUrl('/donate')}>Donate</a></li>
+            </ul>
+          </div>
+          <div className="area-5">
+            {this.renderSocialLinks()}
+            {this.renderMobileLinks()}
+            <p>
+              © 2012–{(new Date().getFullYear())} {siteName} Inc.
+              <br />
+              EdX, Open edX, and MicroMasters are registered trademarks of edX Inc.
```

Comment:
Does this need to differ for White label sites?

Reply:
I'm not sure.